doc/source/examples/multiphysics/tracer-through-cad-junction/tracer-through-cad-junction.rst

```diff
@@ -0,0 +1,161 @@
+    # Mesh
+    #---------------------------------------------------
+    subsection mesh
+        set type                 = gmsh
+        set file name            = vessels.msh
+        set initial refinement   = 0
+        set simplex              = true
+    end
+
+.. warning:: 
+    If the mesh type is set to ``dealii`` instead of ``gmsh``, making sure the imported mesh is built with simplices doesn't make sense; the deal.II generated mesh will be built properly.
+
+.. warning:: 
+    It is crucial to consider a current (2022-01-19) limitation : simplex grid refinement isn't implemented in Lethe.
+
+Setting up the tracer physics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using the tracer physics requires four elements: activating the physics, setting the tracer diffusivity, setting the tracer elements order and setting the boundary conditions. These elements are provided as such:
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+        set tracer = true
+    end
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Physical Properties
+    #---------------------------------------------------
+    subsection physical properties
+        set kinematic viscosity            = 1
+        set tracer diffusivity             = 0.001
+    end
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # FEM
+    #---------------------------------------------------
+    subsection FEM
+        set velocity order            = 1
+        set pressure order            = 1
+        set tracer order              = 1
+    end
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Boundary Conditions
+    #---------------------------------------------------
+    subsection boundary conditions tracer
+    set number                  = 2
+        subsection bc 0
+        set id = 1
+        set type              = dirichlet
+            subsection dirichlet
+                    set Function expression =  0 
+            end
+        end
+        subsection bc 1
+        set id = 2
+        set type              = dirichlet
+            subsection dirichlet
+                    set Function expression =  1 
+            end
+        end
+    end
+
+The boundary conditions are written in a specific way. We have a number of specified 2 boundaries, a Dirichlet condition of 1 for an inlet, and another Dirichlet condition of 0 for the two other inlets; all the remaining boundaries are unspecified. An unspecified boundary condition in Lethe is considered as the natural condition of finite elements, which is a zero Neumann condition. This is the condition needed in that case for the walls and outlets: the walls must not let any tracer out, and the outlets must not apply any constraint on their tracer level.
```

Comment:
adding the difference between the normal boundary condition subsection and the tracer boundary conditions subsection would be helpful I think